### PR TITLE
Feat/apply 2.0

### PR DESCRIPTION
**Pull request by watermelon1024**
# Description

Re-write apply system.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by a3510377 — APPROVED** [review]

File: lipoic/core/models/memberApply.py [hunk 1]
```diff
@@ -28,13 +38,23 @@ class MemberApplyType(TypedDict):
...
+    message_id: int
+    # meeting channel ID
+    channel_id: int
+    # meeting message ID
+    meeting_message_id: int
+    # meeting timestamp
+    meeting_time: int
+    # meeting member ID
+    meeting_member: int
     # email
```
Comment (EvanHsieh0415) [inline lipoic/core/models/memberApply.py:51]: 這裡有需要comment嗎

File: lipoic/core/types/MemberApply.py [hunk 2]
```diff
@@ -27,18 +22,18 @@ class EventData(NamedTuple):
...
     email: Text
-    # self introduction | 自介
+    # self introduction | 自我介紹
     selfIntro: Text
-    # identity | 身分
-    identity: identityType | Text
-    # CV | 簡歷
+    # identity | 您目前的身份?
+    identity: Text
+    # CV | 您的簡歷 (經歷、作品連結等)
```
Comment (EvanHsieh0415) [inline lipoic/core/types/MemberApply.py:29]: `CV` -> `Curriculum Vitae`
這裡沒必要用簡寫

File: lipoic/core/types/MemberApply.py [hunk 3]
```diff
@@ -27,18 +22,18 @@ class EventData(NamedTuple):
 
     # email | 電子郵件
     email: Text
-    # self introduction | 自介
+    # self introduction | 自我介紹
     selfIntro: Text
-    # identity | 身分
-    identity: identityType | Text
-    # CV | 簡歷
+    # identity | 您目前的身份?
```
Comment (EvanHsieh0415) [inline lipoic/core/types/MemberApply.py:27]: comment用直述句，疑問當備註，用`()`括起來，下面略同

**Review by EvanHsieh0415 — COMMENTED** [review]
some question/problem

**Pull request merged**